node/src/chain_spec.rs

```diff
@@ -221,7 +221,7 @@ pub fn testnet_parachain_config(para_id: ParaId) -> Result<ChainSpec, String> {
 				// Pre-funded accounts
 				vec![hex!["30035c21ba9eda780130f2029a80c3e962f56588bc04c36be95a225cb536fb55"].into()],
 				true,
-				para_id,
+				1000.into(),
```

Comment:
all default dev parachain id should be 200 as we're used to, or polkadot launch config.json has to be changed